packages/shared/src/skeletal-loader/TableSkeleton.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Skeleton } from '@patternfly/react-core';
+import './tableSkeleton.scss';
+
+type SkeletalLoaderProps = {
+  // Defaults to 3
+  columns?: number;
+  // Defaults to 5
+  rows?: number;
+};
+export const TableSkeletonLoader: React.FC<SkeletalLoaderProps> = ({
+  columns = 3,
+  rows = 5,
+}) => {
+  return (
+    <>
+      {Array.from({ length: rows }).map((_item1, index1) => (
+        <div>
+          <div className="odf-skeleton-table__skeletal-loader">
+            {Array.from({ length: columns }).map((_item2, index2) => (
+              <Skeleton
+                key={index1 + index2}
+                width="80%"
+                className="odf-skeleton-table-skeletal-loader__item"
```

Comment:
The loader item is still an element, right?
```suggestion
                className="odf-skeleton-table__skeletal-loader-item"
```

Reply:
I think this is correct. `odf-skeleton-table-skeletal-loader` would be the block and `item` would be element here.

Reply:
Then you're not following the same pattern as per the above class `odf-skeleton-table__skeletal-loader` it's clear that the common block is `odf-skeleton-table`. It's better to follow a consistent pattern.

Reply:
my bad updating